src/api.rs

```diff
@@ -429,6 +448,8 @@ pub struct MasteringDisplay {
     pub min_luminance: u32,
 }
 
+/// Contain details like the maximum content light level which is the nit value corresponding to the brightest pixel used anywhere in the frame.\
```

Comment:
either use `contains` here, or use `contain` everywhere, consistency is important for documentation
also extra `\`